components/shapediver/ui/DrawingOptionsComponent.tsx

```diff
@@ -107,51 +110,38 @@ export default function DrawingOptionsComponent(props: {
 		}
 	}, [drawingToolsApi]);
 
-
-
 	/**
 	 * The description of the drawing tools.
 	 * This description is shown when hovering over the info button.
 	 */
-	const description =
-		<List listStyleType="none" fw={400} size="xs">
-			<List.Item>Adding Points:
-				<List listStyleType="disc" fw={400} size="xs" fs="italic">
-					<List.Item>Starts automatically if no points exist</List.Item>
-					<List.Item>Press Insert to add a point at cursor position</List.Item>
-					<List.Item>Hover over a line segment to add a new point at the center</List.Item>
-				</List>
-			</List.Item>
-			<List.Item>Removing Points:
-				<List listStyleType="disc" fw={400} size="xs" fs="italic">
-					<List.Item>Select points and press Delete</List.Item>
-				</List>
-			</List.Item>
-			<List.Item>Moving Points:
-				<List listStyleType="disc" fw={400} size="xs" fs="italic">
-					<List.Item>Drag individual points or select multiple points to move together</List.Item>
-					<List.Item>Movement restrictions:
-						<List listStyleType="disc" fw={400} size="xs" fs="italic">
-							<List.Item>Press g for grid</List.Item>
-							<List.Item>Press a for angles</List.Item>
-							<List.Item>Press x/y/z for axes</List.Item>
-						</List>
-					</List.Item>
-				</List>
-			</List.Item>
-			<List.Item>History of Operations:
-				<List listStyleType="disc" fw={400} size="xs" fs="italic">
-					<List.Item>Press Ctrl+z to undo</List.Item>
-					<List.Item>Press Ctrl+y to redo</List.Item>
-				</List>
-			</List.Item>
-			<List.Item>Update/Cancel:
-				<List listStyleType="disc" fw={400} size="xs" fs="italic">
-					<List.Item>Press Confirm or Enter to send changes to the server</List.Item>
-					<List.Item>Press Cancel or Escape to discard changes</List.Item>
-				</List>
-			</List.Item>
-		</List>;
+	const markdown = 
+`- Adding Points:
+  - *Starts automatically if no points exist*
+  - *Press Insert to add a point at cursor position*
+  - *Hover over a line segment to add a new point at the center*
+
+- Removing Points:
+  - *Select points and press Delete*
+
+- Moving Points:
+  - *Drag individual points or select multiple points to move together*
+  - Movement restrictions:
+    - *Press g for grid*
+    - *Press a for angles*
+    - *Press x/y/z for axes*
+
+- History of Operations:
+  - *Press Ctrl+z to undo*
+  - *Press Ctrl+y to redo*
+
+- Update/Cancel:
+  - *Press Confirm or Enter to send changes to the server*
+  - *Press Cancel or Escape to discard changes*
+`;
+
+
+	// define the size of the components
+	const size: MantineSize = "xs";
```

Comment:
@MajorMeerkatThe3rd please create a follow up App Builder task for me to plug the theme as discussed. I think we can get the size from the theme as well, let's see.

Reply:
@snabela [here you go](https://shapediver.atlassian.net/browse/SS-8092)